es6-classes/circle.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super();
+    this.radius = radius;
+    this.area = this.radius * this.radius * Math.PI;
+    this.perimeter = this.radius * Math.PI * 2;
```

Comment:
You should not be creating these properties manually. You should be utilizing the `super` function to create these

Reply:
Thank you Shawn! Updated code and repushed. Let me know if they are correct!